src/app/app-routing.module.ts

```diff
@@ -15,6 +18,9 @@ export const routerConfig: Routes = [
   { path: 'validation', component: ValidationComponent },
   { path: 'statistiques', component: StatistiquesComponent },
   { path: 'publications', component: PublicationsComponent },
+  { path: 'homestudents', component: HomestudentsComponent },
```

Comment:
All english or all french but no mix

Reply:
alright fam gotta do some more refacto then

Reply:
btw the app was already mixed since there was a "home" route and "statistiques"

Reply:
maybe just change "Statistiques" 😂